### PR TITLE
First branch

### PR DESCRIPTION
**Pull request by GraceHtet**
to view my first pull request

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#node_modules
+.github/workflow/linters.yml
```
Comment (franclobo) [inline .gitignore]: - [ ] Please 🙏🏽 quit this folder from the .gitignore file. The linter should stay in the root folder.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+test.md
```
Comment (franclobo) [inline .gitignore]: - [ ] Kindly correct the path to the `test.md` file, we could use as follow 👉🏽 `./test.md` (**with a dot** at the beginning)

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#node_modules
```
Comment (franclobo) [inline .gitignore]: - [ ] Kindly correct the path to the `node_modules` folder, we could use as follow 👉🏽 `/node_modules` (**without a dot** at the beginning)

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @GraceHtet, 

Good job so far!
You did it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Correct Github flow ✔️ 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Kindly all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo.
- [ ] Please 🙏🏽 remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
  - Kindly writhe a descriptive title and list the activities in your pull request 👇🏽 

| Your PR | Expected PR |
|---|---|
| image | image |

- [ ] Please follow these [steps](https://github.com/microverseinc/linters-config/tree/master/html-css) to install the linter and make sure that all of them are checked.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

## **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GraceHtet** [comment]
Fixed some mistakes in .gitignore
Added some contents to readme.md

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED  
 ## Hi @GraceHtet 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Your repo is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 * Use new readme template :white_check_mark: 
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers four_leaf_clover:


 
 ## Optional suggestions

- [ ] Kindly make sure to document your README professionally. Please use the [Microverse README template](https://github.com/microverseinc/readme-template) and customize it. as required by the previous review
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — COMMENTED** [review]
Hi @GraceHtet, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, with hello-microverse PR but this is a `: Portfolio: setup and mobile first` re-review project. Please submit the correct URL.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**